app/containers/Achievements/Achievement.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import achievementReached from 'images/achievement_reached.svg?inline';
+import achievementNotReached from 'images/achievement_not_reached.svg?inline';
+
+import Span from 'components/Span';
+
+const Img = styled('img')`
+  margin-right: 15px;
+`;
+
+const TitleBlock = styled(Span)`
+  display: block;
+`;
+
+const DescriptionBlock = styled(TitleBlock)`
+  margin-top: 15px;
+`;
+
+const Bage = styled.div`
+  display: flex;
+`;
+
+const ValueSpan = styled.span`
+  font-size: 18px;
+  margin-right: 4px;
+`;
+
+const Achievement = ({
+  reached,
+  title,
+  multipleTitle,
+  description,
+  value,
+  id,
+}) => {
+  return (
+    <Bage>
+      {reached && <Img src={achievementReached} alt="reached achievement" />}
+      {!reached && (
+        <Img src={achievementNotReached} alt="not reached achievement" />
+      )}
+      <div>
+        <TitleBlock>
+          <strong>
+            <ValueSpan>{(id === 1 || id === 2 || id === 3) && value}</ValueSpan>
```

Comment:
Use constants instead of these magic numbers.

Reply:
Can setup array in constants and then use `includes(value)` to check that it is one of a values.

Reply:
badgesWithValuesIds array added for detecting badges with values